Tasks/PowerShellV2/Strings/resources.resjson/de-DE/resources.resjson

```diff
@@ -13,7 +14,15 @@
   "loc.input.help.arguments": "An das PowerShell-Skript übergebene Argumente. Entweder Ordnungszahlparameter oder benannte Parameter.",
   "loc.input.label.script": "Skript",
   "loc.input.label.errorActionPreference": "ErrorActionPreference",
+  "loc.input.label.warningPreference": "WarningPreference",
+  "loc.input.label.informationPreference": "InformationPreference",
+  "loc.input.label.verbosePreference": "VerbosePreference",
+  "loc.input.label.debugPreference": "DebugPreference",
   "loc.input.help.errorActionPreference": "Stellt oben in Ihrem Skript die Zeile \"$ErrorActionPreference = 'VALUE'\" voran.",
+  "loc.input.help.warningPreference": "Stellt oben in Ihrem Skript die Zeile \"$WarningPreference = 'VALUE'\" voran.",
```

Comment:
Could you please remove updates for localized files and leave English only - since we have separate flow for translation?